app/react/Forms/components/MultiSelect.js

```diff
@@ -215,6 +215,15 @@ export default class MultiSelect extends Component {
     );
   }
 
+  renderInfoForEmptyOptions() {
+    const { sourceName } = this.props;
+    const info = sourceName ? `No items of type '${sourceName}' found.` :
```

Comment:
This text should be translated using the <Translate> component, but also the var sourceName is the name of a thesaurus that has its own translation. 
Is `No items of type '${sourceName}' found.` a requirement of the issue? If not I'll suggest to remove it so we reduce complexity in the component and also avoid the problem with the translation.

Reply:
I don't think it's a hard requirement, but it was specified in the issue. Without it, it may not be immediately obvious to the user which thesauri or template the multiple options are coming from until they go check in the settings.
But I can change it back to `No options found` if the convenience is not worth the complexity.

Reply:
I've changed it to 'No options found'.